app/services/changePassword.service.ts

```diff
@@ -27,6 +30,10 @@ const changePassword = async (oldPassword: string, newPassword: string): Promise
     const contacts = await decryptContacts(store.state.secretKey);
     store.dispatch(updatePassphraseAction(newEncryptedPassphrase, secretKey));
     updateContacts(contacts);
+
+    await ipcRenderer.invoke('start-full-service', oldPassword, newPassword);
+    await new Promise((resolve) => setTimeout(resolve, 5000));
```

Comment:
once more, question about these 5 seconds

Reply:
waiting for full service to start